### PR TITLE
Add my file2

### PR DESCRIPTION
**Pull request by jfernan2**

### CODE REVIEW

File: bin/Powheg/production/TT_hdamp_TuneT4_NNPDF30_8TeV_powheg/TT_hdamp_TuneT4_NNPDF30_8TeV_powheg.input [hunk 1]
```diff
@@ -0,0 +1,79 @@
+! TTbar production parameters
```
Comment (perrozzi) [inline bin/Powheg/production/TT_hdamp_TuneT4_NNPDF30_8TeV_powheg/TT_hdamp_TuneT4_NNPDF30_8TeV_powheg.input]: is this file the same as https://github.com/cms-sw/genproductions/pull/976 ?

**Comment by jfernan2** [comment]
Yes, sorry, by mistake I pulled it twice
Thanks

File: bin/Powheg/runcmsgrid_powheg_hdampgrid.sh [hunk 2]
```diff
@@ -61,7 +61,7 @@ mkdir ${myDir}; cd ${myDir} ;
 #ln -s `which gfortran` g77
```
Comment (perrozzi) [inline bin/Powheg/runcmsgrid_powheg_hdampgrid.sh]: can you elaborate a bit on the changes implemented to this file?

**Comment by perrozzi** [comment]
can you remove the unnecessary file? can you also edit the name of the PR? thanks

**Comment by jfernan2** [comment]
If you don't mind I'll close this one and start a new one because I don't want to screw up things..

**Pull request closed without merging**